src/fileFrom/UploadFromUrl.ts

```diff
@@ -53,10 +54,29 @@ export class UploadFromUrl extends UploadFrom {
   }
 
   private handleFromUrlStatusResponse = (token: string, response: FromUrlStatusResponse) => {
+    this.isFileUploadedFromUrlPolling = checkFileIsUploadedFromUrl({
+      token,
+      timeout: 1000,
```

Comment:
This is too few. Should be ten seconds, at least.

Reply:
I agree